compare-bug-version-and-git-branch.sh

```diff
@@ -42,18 +42,24 @@ while [ -z "${BUG_PRODUCT}" -a ${BZQTRY} -le 3 ]; do
         echo 1
     fi
 
-    BZQOUT=$(bugzilla ${BZQOPTS} query -b ${BUG} --outputformat='%{product}:%{version}:%{groups}:%{status}')
+    BZQOUT=$(bugzilla ${BZQOPTS} query -b ${BUG} --outputformat='%{product}:%{version}:%{groups}:%{status}:%{assigned_to}')
     BUG_PRODUCT=$(cut -d: -f1 <<< "${BZQOUT}")
     BUG_VERSION=$(cut -d: -f2 <<< "${BZQOUT}")
     BUG_GROUPS=$(cut -d: -f3 <<< "${BZQOUT}")
     BUG_STATUS=$(cut -d: -f4 <<< "${BZQOUT}")
+    BUG_OWNER=$(cut -d: -f5 <<< "${BZQOUT}")
 done
 
 if [ "${BUG_STATUS}" != "NEW" -a "${BUG_STATUS}" != "POST" -a "${BUG_STATUS}" != "ASSIGNED" ]; then
     echo "BUG id ${BUG} has an invalid status as ${BUG_STATUS}. Acceptable status values are NEW, ASSIGNED or POST."
     exit 1
 fi
 
+if [ "${BUG_OWNER}" = "bugs@gluster.org" ]; then
+    echo "BUG id ${BUG} has an invalid owner"
```

Comment:
Could you change this message to: "BUG id ${BUG} is owned by bugs@gluster.org. Please assign it to yourself. If you do not have the right permissions, please ask the component owner for help".